doc/source/conf.py

```diff
@@ -230,7 +230,7 @@
 # the API pages as top-level functions) based on a template (GH9911)
 moved_api_pages = [
     'pandas.core.common.isnull', 'pandas.core.common.notnull', 'pandas.core.reshape.get_dummies',
-    'pandas.tools.merge.concat', 'pandas.tools.merge.merge', 'pandas.tools.pivot.pivot_table',
```

Comment:
just leave it as is. This is for "back compat" with existing links on the internet, as we previously included the docstring pages for concat using those paths, but now just as `pandas.concat`